Testing/functional/tests/cypress/db/seed.sql

```diff
@@ -17,6 +17,7 @@ TRUNCATE gateway."Dependent" CASCADE;
 TRUNCATE gateway."AgentAudit" CASCADE;
 TRUNCATE gateway."BlockedAccess" CASCADE;
 TRUNCATE gateway."Outbox" CASCADE;
+TRUNCATE gateway."Delegation" CASCADE;
```

Comment:
This may not be needed as there is already a Cascade on ResourceDelegate.

Reply:
CASCADE wouldn't be enough There are scenarios where there is a record in delegation but not ResourceDelegate.